roles/pulsar-post-tasks/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Install slurm-drmaa
+  package:
+    name: slurm-drmaa1
+    state: present
+- name: own the /mnt/pulsar dirs as ubuntu
+  file:
+      path: /mnt/pulsar
+      owner: ubuntu
+      group: ubuntu
+      state: directory
+      recurse: yes
+      mode: 0754
+- name: own the /mnt/galaxy dirs as ubuntu
```

Comment:
same here

Reply:
I've added this as `galaxy_dir` in pulsarservers.yml.  I didn't want it to clash with galaxy_root from group_vars/galaxyservers.yml even though it is not in scope here